src/notation/internal/notation.h

```diff
@@ -36,7 +36,7 @@ class NotationInteraction;
 class NotationPlayback;
 class Notation : virtual public INotation, public IGetScore, public async::Asyncable
 {
-    INJECT(notation, INotationConfiguration, configuration)
+    INJECT_STATIC(notation, INotationConfiguration, configuration)
```

Comment:
Seems that INJECT_STATIC is no longer needed

Reply:
Now needed, used in static method Notation::init()